projects/libs/core-library-typescript/test/block-storage/block-storage.spec.ts

```diff
@@ -117,4 +121,33 @@ describe("Block storage", () => {
       await expect(blockStorage.getBlocks(lastBlock.header.height)).to.eventually.be.eql([exampleBlock]);
     });
   });
+
+  describe("run checkStartupStatus", () => {
+    it("checkStartupStatus should succeed", async () => {
+      const startupStatus = await blockStorage.startupCheck();
+      return expect(startupStatus).to.deep.equal(<StartupStatus>{ name: "block", status: STARTUP_STATUS.OK });
+    });
+  });
+});
+
+describe("Block storage - Bad setup (no transaction pool)", () => {
+  let blockStorage: BlockStorage;
+  // let lastBlock: types.Block;
+
+  beforeEach(async () => {
+    try {
+      fsExtra.removeSync(LEVELDB_PATH);
+    } catch (e) { }
+    blockStorage = await initBlockStorage(undefined);
```

Comment:
Why do you init with `undefined`?

Reply:
On purpose, to init an invalid value so a test can fail